### DISCUSSION TITLE
Figma Design

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#4] [dhaqsoPay Extension Developers]
@dugsiiyeinc/dhaqsopay-extension-developers design-ka waad bilowdeen filaa ?

excalidraw link:
https://excalidraw.com/#json=uThYIUVh84DXkPOomBxnt,12_BnUrtjcymjo2FDQv-KQ

**Comment by iftiiin** [comment]
Haa eng

**Comment by iftiiin** [comment]
![extension](https://github.com/user-attachments/assets/9ba36b1a-eb9a-4962-8754-aa0e222d5013)
asxaabta design-kaan sidee waaye.
adinkana soo gudbiya i.a

**Reply by AishaKeyf** [reply]
asc, eng qaybta hoose in radius ka laga saaro oo caddaanka lagu fidiyo baa ila wacan .
yacnii inta buluugaa in laga tirtiro.

**Reply by ducesane** [reply]
Design fican waaye

**Reply by sharafdin** [reply]
> ![extension](https://private-user-images.githubusercontent.com/105747964/385935147-9ba36b1a-eb9a-4962-8754-aa0e222d5013.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE1ODI2MTEsIm5iZiI6MTczMTU4MjMxMSwicGF0aCI6Ii8xMDU3NDc5NjQvMzg1OTM1MTQ3LTliYTM2YjFhLWViOWEtNDk2Mi04NzU0LWFhMGUyMjJkNTAxMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExNFQxMTA1MTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYWM0MTliMGI1ZDc4MWQwYTE3MzM4MzFlNmMzMmYwMTI0ZGU0OTUzMmRlNTA0OTZlNWYwMmQ4NTFmMjkzMWY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.gURfm18J4k3FBskWWWodqLOG8sVWP8w9198bOfIfeT0) asxaabta design-kaan sidee waaye. adinkana soo gudbiya i.a

eng isfitin ma xumo lkn another option ina siiya oo design kale soo habeeya Insha'Allah! @dugsiiyeinc/dhaqsopay-extension-developers team adinkana design kale soo dira bal Insha'Allah!

**Comment by hayle01** [comment]
Waxaan soo diri doona caawa design-ka extension-ka haddi uu ina anfaco waan qaadanaynaa

**Reply by sharafdin** [reply]
Insha'Allah! Akhii
